CHANGELOG.md

```diff
@@ -1,5 +1,94 @@
 # Ansible Collections for HPE OneView Change Log
 
+## v6.2.0
```

Comment:
Add unreleased tag.. Also modify the workflow file.

Reply:
Updated Changelog, i will update workflow file as part of release PR.